Formula/ndenv.rb

```diff
@@ -0,0 +1,41 @@
+class Ndenv < Formula
+  desc "Node version manager"
+  homepage "https://github.com/riywo/ndenv"
+  url "https://github.com/riywo/ndenv/archive/v0.4.0.tar.gz"
+  sha256 "1a85e4c0c0eee24d709cbc7b5c9d50709bf51cf7fe996a1548797a4079e0b6e4"
+  head "https://github.com/riywo/ndenv.git"
+
+  depends_on "node-build" => :recommended
+
+  def install
+    inreplace "libexec/ndenv" do |s|
```

Comment:
```
if HOMEBREW_PREFIX.to_s != "/usr/local"
  inreplace "libexec/ndenv", ":/usr/local/etc/ndenv.d", ":#{HOMEBREW_PREFIX}/etc/ndenv.d\\0"
end
```

Also, it's worth requesting to upstream to provide a dedicated `configure` or similar to handle this rather than our patching.